cmd/microcloud/ask.go

```diff
@@ -732,6 +727,10 @@ func (c *initConfig) askRemotePool(sh *service.Handler) error {
 
 			selectedDisks = targetDisks
 
+			if insufficientDisks {
+				return fmt.Errorf("Disk configuration does not meet recommendations for fault tolerance. At least %d systems must supply disks.", recommendedDiskSpread)
```

Comment:
In case of non-interactive I think the error message should also mention that this it not recommended?

Reply:
Good point, I'll add a log to the preseed path.